app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError, VaccineError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    count_not_wear_mask = 0
+    for person in friends:
+        try:
+            cafe.visit_cafe(person)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            count_not_wear_mask += 1
+
+    if count_not_wear_mask > 0:
```

Comment:
You don't need to compare with 0 here, just use 
`if count_not_wear_mask:` as it was told in checklist